### PR TITLE
fix(seed): Space between function name and parameters was not

### LINKED ISSUE #16304: Functional Programming: Pass Arguments to Avoid... - Correct answer not accepted
#### Challenge Name

Functional Programming: Pass Arguments to Avoid External Dependence in a Function

#### Issue Description

Correct answer is not accepted when space between function name and params is not removed.

#### Browser Information

* Browser Name, Version: Chrome  63.0.3239.84 (Build oficial) (64 bits)
* Operating System: Win10
* Mobile, Desktop, or Tablet:  Desktop 

#### Your Code

Not accepted:
```js
function incrementer (val) {
  return val+1;
  // Add your code above this line
}
```
Accepted:
```js
function incrementer(val) {
  return val+1;
  // Add your code above this line
}
```

#### Screenshot

fireshot capture 162 - learn to code i freecodecamp_ - https___beta freecodecamp org_en_c

fireshot capture 164 - learn to code i freecodecamp_ - https___beta freecodecamp org_en_c

### PR DESCRIPTION
**Pull request by sagarjhaa**
the regular expression has been changed in test to check for zero or
more space.

Closes #16304

#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16304 

#### Description

Edited the regular expression on second test to check for zero to many space between function name and the parameters with "\s*". I have tested with zero space, 1 space and many space between function name and parameters and it works for all the cases.

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/functional-programming.json [hunk 1]
```diff
@@ -289,7 +289,7 @@
       ],
       "tests": [
           "assert(fixedValue === 4, 'message: Your function <code>incrementer</code> should not change the value of <code>fixedValue</code>.');",
-          "assert(code.match(/function\\s+?incrementer\\(.+?\\)/g), 'message: Your <code>incrementer</code> function should take a parameter.');",
+          "assert(code.match(/function\\s+?incrementer\\s*\\(.+?\\)/g), 'message: Your <code>incrementer</code> function should take a parameter.');",
```
Comment (Bouncey) [inline seed/challenges/02-javascript-algorithms-and-data-structures/functional-programming.json]: Can you make this a lazy match please with the `*?` quantifier

i.e. `incrementer\\s*?\\(`

**Review by Bouncey — CHANGES_REQUESTED** [review]

**Comment by sagarjhaa** [comment]
Sure I will make the change.

**Comment by sagarjhaa** [comment]
@camperbot I have made changes.

**Comment by sagarjhaa** [comment]
@raisedadead do I need to do anything in order to get this pull request merged?

**Comment by raisedadead** [comment]
Nope, we will pull down and review.. thanks for contributing.

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **AWESOME! LGTM.** :sparkles: :sparkles: 

Thanks @sagarjhaa , for this pull request and contributing to the code-base for the first time.

We will be looking forward for more contributions like this.

Cheers and happy contributing!

**Comment by raisedadead** [comment]
P.S: @camperbot is just a dumb robot, doesn't do much other than ranting, I should add some humour settings to it.

**Pull request merged**